test/settings.py

```diff
@@ -111,3 +111,5 @@
 COMMENT_PER_PAGE = 4
 
 COMMENT_USE_GRAVATAR = True
+
+COMMENT_ALLOW_TRANSLATION = True
```

Comment:
What the purpose of making settings' variable? Does it harm if the `JavaScriptCatalog` path is always included?

Reply:
The path can be always included. The script tag to include the `javascript` file inside [`static.html`](https://github.com/Radi85/Comment/pull/88/files#diff-ac8dcc63ef631044a84d92a272230469R4) should only be included when translations are enabled. This will avoid unnecessary overhead. I will make the change.